src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $month-width;
+  padding: 10px;
+
+  @for $days-count from 28 through 31 {
+    &--month-length--#{$days-count} {
+      & :nth-child(n + #{ $days-count + 1 }) {
+        display: none;
+      }
+    }
+  }
+
+  @each $day-name, $first-day, $weekend  in $start-days {
+    &--start-day--#{$day-name} {
+      & :first-child {
+        margin-left: $first-day + $first-day * $day-sides;
+      }
+    }
+  }
+
+  &__day {
+    height: $day-sides;
+    width: $day-sides;
+    background-color: $day-color;
+    border: 1px solid black;
```

Comment:
use variables for colors

Reply:
don't like to use variables for default colors)

Reply:
Usually there are few colours on the project (for example 10 or less) and almost all of them are different from default. At some time designers can decide to change them a bit, and variables will make this changes easily, because it's painful to find every particular color usage to change it 🙂  That's why we recommend to use variables even in small projects (to get used to this logic)